controller/handler_edge_ctrl/remove_tunnel_terminator.go

```diff
@@ -61,12 +61,14 @@ func (self *removeTunnelTerminatorHandler) RemoveTerminator(ctx *RemoveTunnelTer
 		return
 	}
 
-	ctx.verifyTerminator(ctx.terminatorId, edge_common.TunnelBinding)
+	t := ctx.verifyTerminator(ctx.terminatorId, edge_common.TunnelBinding)
 	if ctx.err != nil {
 		self.returnError(ctx, ctx.err)
 		return
 	}
 
+	logger = logger.WithField("serviceId", t.Service)
```

Comment:
Making sure this is actually correct. `Service` = "serviced"

Reply:
Yes, double checked.